lab-kayla/.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
**TODO FOR LOST POINTS**

This is a top level comment that you need to do your README, this is an easy 2 points.